### ISSUE TITLE
Flatbuffers for serialisation

### ISSUE THREAD

**Issue by tekjar** [pingcap/grpc-rs#265]
**Is your feature request related to a problem? Please describe.**
Streaming/RPC using flatbuffers might be a lower overhead option to protobuf. Is this support planned?

**Comment by ice1000** [comment]
Not now, but thanks for your suggestion.
We have a `Read`/`Write` trait support for reducing data copying, though.

**Comment by siddontang** [comment]
Seem we can already support using another codec? @BusyJay

**Comment by BusyJay** [comment]
Yes, although we support protobuf out of box, but you are free to add/implement other codec support. See also #155.
